### PR TITLE
Bump supported versions to include PHP8.2, 8.3

### PR DESCRIPTION
**Pull request by braindawg**
## Description

Adds PHP 8.2 and 8.3 to the CI test suite and README.

Updated the version constraint on PHP in Composer to explicitly list the supported PHP8 versions.

## CHANGELOG

* [CHANGED] Explicitly support PHP 8.2 and 8.3

### CODE REVIEW

**Comment by SuperDJ** [comment]
You might also want to add PHP8.3 https://www.php.net/supported-versions.php

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "keywords": ["php-pusher-server", "pusher", "rest", "realtime", "real-time", "real time", "messaging", "push", "trigger", "publish", "events"],
     "license": "MIT",
     "require": {
-        "php": "^7.3|^8.0",
+        "php": "^7.3|^8.0|^8.1|^8.2",
```
Comment (GrahamCampbell) [inline composer.json:7]: `|^8.1|^8.2` is incorrect. `^8.0` already allows all 8.x.

Comment (GrahamCampbell) [reply composer.json:7]: i would recommend just reverting the change to the composer.json. people can install it on any php 8 version, at their own risk. that's super common.

**Comment by braindawg** [comment]
Reverted unnecessary Composer changes and added PHP 8.3 to the list as suggested by @SuperDJ. May as well test against all current versions.

**Review by aonemd — APPROVED** [review]

**Comment by aonemd** [comment]
Thanks for opening this, @braindawg 
It's merged now!

**Pull request merged**